pkg/cmd/oauth-server/cmd.go

```diff
@@ -51,24 +55,30 @@ func NewOsinServer(out, errout io.Writer, stopCh <-chan struct{}) *cobra.Command
 		},
 	}
 
+	// Handle Flags
 	flags := cmd.Flags()
-	// This command only supports reading from config
-	flags.StringVar(&options.ConfigFile, "config", "", "Location of the osin configuration file to run from.")
+	opts.Audit.AddFlags(flags)
```

Comment:
I created a general `AddFlags` method for options, but I would've needed to add the `cmd` in there, so I left it as is.